be/src/storage/memtable.cpp

```diff
@@ -24,6 +24,19 @@ namespace starrocks::vectorized {
 static const string LOAD_OP_COLUMN = "__op";
 static const size_t kPrimaryKeyLimitSize = 128;
 
+bool MemTable::_is_aggregate_needed() {
+    return _keys_type != KeysType::DUP_KEYS;
+}
+
+void MemTable::_init_aggregator_if_needed() {
+    if (_is_aggregate_needed()) {
```

Comment:
There is not necessary to define a new function named _is_aggregate_needed().

Reply:
why? otherwise judge if key type is DUP_KEYS here and there, this is a detail.